app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -435,4 +434,8 @@ def targeted_persister_klass
   def parser_klass
     @parser_klass ||= ems.class::Inventory::Parser
   end
+
+  def saver_klass
+    @saver_klass ||= self.class.module_parent::Saver
```

Comment:
Not sure why I cached these class names...wonder how much time is actually saved doing this

Reply:
Was just trying to be "cute" and save some columns but they're just hard-coded constants